src/main/java/mate/academy/bookstore/dto/shopping/cart/item/CartItemRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.bookstore.dto.shopping.cart.item;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class CartItemRequestDto {
+    @Positive
+    @NotBlank
```

Comment:
Is this annotation applicable to Long?

Reply:
https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/javax/validation/constraints/Positive.html
![зображення_2024-01-03_160257383](https://github.com/AgroFix/book-store/assets/89842485/d7594da8-8b8d-442c-89e2-8a74170c698c)

Reply:
I've tried to run your app and have this error when adding item:
```
jakarta.validation.UnexpectedTypeException: HV000030: No validator could be found for constraint 'jakarta.validation.constraints.NotBlank' validating type 'java.lang.Long'. Check configuration for 'bookId'
```
Did you test your endpoints? Because they are not working...